inc/TRestRawSignal.h

```diff
@@ -56,7 +52,7 @@ class TRestRawSignal : public TObject {
     TGraph* fGraph;  //!
 
     /// A std::vector containing the index of points that are identified over threshold.
-    std::vector<Int_t> fPointsOverThreshold;  //!
+    std::vector<std::pair<Float_t, Float_t> > fPointsOverThreshold;  //!
```

Comment:
Better to have `fPointsOverThreshold` in correspondence with TRestRawSignal's data structure. x should be time bin(int) and y should be amplitude(unsigned short). Therefore the returned type should be `std::vector<std::pair<Int_t, UShort_t> >`

Reply:
Yes, I didn't notice this, I also think it would make more sense `<Int_t, UShort_t>`

Reply:
Looking at the code `fPointsOverThreshold` corresponds to the points with the baseline substracted, so for me it doesn't make sense to return `std::vector<std::pair<Int_t, Short_t> >` since we will be loosing precision.